docs/constants/navigation-data.js

```diff
@@ -31,6 +31,7 @@ const DIR_MAPPING = {
   'app-signing': 'App Signing',
   'build-reference': 'Reference',
   submit: 'EAS Submit',
+  'update-spec': 'Technical Spec',
```

Comment:
This specification describes Expo Updates, not EAS Update. It belongs in the Expo section of the docs, not the EAS section.

expo-updates is an implementation of an Expo Updates client. EAS Update is an implementation of an Expo Updates server.

The terms expo-updates and EAS Update should not appear in this specification except when referred to as examples.

Reply:
Will do.